src/main/java/com/ezylang/evalex/data/EvaluationValue.java

```diff
@@ -310,6 +348,44 @@ public Boolean getBooleanValue() {
     }
   }
 
+  /**
+   * Gets a {@link Instant} representation of the value. If possible and needed, a conversion will
+   * be made.
+   *
+   * <ul>
+   *   <li>Any non-zero number value will return the instant from the epoc value.
```

Comment:
I think this should return something like Instant.ofEpochMilli(((BigDecimal) value).longValue())